packages/common/src/ui/components/menus/DropdownMenu/DropdownMenu.tsx

```diff
@@ -94,9 +95,11 @@ export const DropdownMenu: FC<PropsWithChildren<DropdownMenuProps>> = ({
         disabled={disabled}
         value=""
         size="small"
+        open={open}
         labelId={`action-drop-down-label-${label}`}
         variant="outlined"
         IconComponent={ChevronDownIcon}
+        onClick={() => setOpen(curr => !curr)}
```

Comment:
The drop-down was staying open after clicking it. This is the only way I could find to change that -- perhaps there is a better way?

Reply:
will have to investigate. I'd hope that the component would provide something better

Reply:
Hmm, there's something about the `<DropdownMenu/>` component that is making the `<Select>`component behave in a non-standard way. When I replace the menu with regular Mui `<MenuItem>` components it works as expected. Will investigate further.

Reply:
Objection withdrawn.. I've played around and realised that we're using a `Select` component and not setting the value, so not using it as a Select. Should probably be using a [Menu](https://mui.com/material-ui/react-menu/) which requires managing the open state manually anyway. So.. back to where we started with what you've already done!